webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/index.js

```diff
@@ -31,12 +31,8 @@ const Controller = ({
   storagePodsStatus,
   storagePodsError,
 }) => {
-  const getEventValue = e => {
-    if (!e.target) {
-      return e;
-    }
-    return e.target.type === 'checkbox' ? e.target.checked : e.target.value;
-  };
+  const getEventValue = e =>
+    e.target.type === 'checkbox' ? e.target.checked : e.target.value;
```

Comment:
nit: could you change it into something like: 
`const getEventValue = ( { target: { type, checked, value } } ) =>  type === 'checkbox' ? checked : value;`

Reply:
thanks, I have moved that one to the helper function for the new event on inputs.